### PR TITLE
update descriptions for What's New in V5

### PR DESCRIPTION
**Pull request by LaureBrosseau**

### CODE REVIEW

File: content/updates/index.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
     "img": "illus-automation-v5.svg",
     "eeOnly": true
   },
   "governance": {
     "title": "Governance",
-    "description": "Data governance is key to further ensure the ease of enrichment and sustainability of your product catalog. That's why we added lots of great features to make our product modelling capabilities even more flexible and powerful. Among others, you'll now be able to enjoy new associations types, a revamped Yes/No attribute and enhancements around product variants.",
+    "description": "Data governance is essential to further guarantee the ease of enrichment and the sustainability of your product catalog. That’s why we’ve added a lot of cool features to make our product modeling capabilities even more flexible and powerful. Among other things, you will now be able to enjoy new associations types, a revamped Yes/No attribute, and enhancements around product variants.",
```
Comment (NolwennP) [inline content/updates/index.json:10]:
```suggestion
    "description": "Data governance is essential to further guarantee the ease of enrichment and the sustainability of your product catalog. That’s why we’ve added a lot of cool features to make our product modeling capabilities even more flexible and powerful. Among other things, you will now be able to enjoy new associations types, a revamped Yes/No attribute, and enhancements around variant products.",
```

File: content/updates/index.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
 {
   "automation": {
     "title": "Automation",
-    "description": "In your daily enrichment tasks, you may sometimes have the feeling of wasting your time in repetitive and mundane tasks. That's where the enhancements we did this year around automation can help you! You can now enjoy our brand new Rules builder, as well as other Rules engine improvements.",
+    "description": "In your daily work tasks, you can sometimes feel that you have too many repetitive and mundane tasks. This is where the improvements we've made this year around automation will help! You can now take advantage of our brand new Rules Builder, as well as other improvements to the Rules Engine.",
```
Comment (camillebourjade) [inline content/updates/index.json:4]:
```suggestion
    "description": "In your daily work, you can sometimes feel that you have too many repetitive and mundane tasks. This is where the improvements we've made this year around automation come in handy! You can now take advantage of our brand new Rules Builder, as well as other improvements to the Rules Engine.",
```

File: content/updates/index.json [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
     "img": "illus-automation-v5.svg",
     "eeOnly": true
   },
   "governance": {
     "title": "Governance",
-    "description": "Data governance is key to further ensure the ease of enrichment and sustainability of your product catalog. That's why we added lots of great features to make our product modelling capabilities even more flexible and powerful. Among others, you'll now be able to enjoy new associations types, a revamped Yes/No attribute and enhancements around product variants.",
+    "description": "Data governance is essential to further guarantee the ease of enrichment and the sustainability of your product catalog. That’s why we’ve added a lot of cool features to make our product modeling capabilities even more flexible and powerful. Among other things, you will now be able to enjoy new associations types, a revamped Yes/No attribute, and enhancements around variant products.",
```
Comment (camillebourjade) [inline content/updates/index.json:10]:
```suggestion
    "description": "Data governance is essential to further guarantee the ease of enrichment and the sustainability of your product catalog. That’s why we’ve added a lot of cool features to make our product modeling capabilities even more flexible and powerful. Among other things, you are now able to enjoy new associations types, a revamped Yes/No attribute, and enhancements around variant products.",
```

**Review by camillebourjade — APPROVED** [review]
English fix

**Pull request merged**